src/App.tsx

```diff
@@ -0,0 +1,18 @@
+// import logo from './logo.svg';
+import React from 'react';
+import './App.css';
+import Header from './components/Header';
+import Carousel from './components/Carousel';
+
+function App() {
+  return (
+    <div className="App">
+      <Header></Header>
+      <Carousel text="MOVIES" type="movie" />
```

Comment:
Please create a separate folder for constants. You could put static text in it. It makes code more reusable.
eg.
```
export const CAROUSEL_TITLES = {
MOVIES: "MOVIES",
TV_SHOWS: "TV SHOWS",...
}
```